src/github.com/getlantern/flashlight/genconfig/genfallbacks.go

```diff
@@ -0,0 +1,256 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"sync"
+	"text/template"
+
+	"github.com/getlantern/flashlight/client"
+	"github.com/getlantern/golog"
+	"github.com/getlantern/yaml"
+)
+
+const (
+	numberOfWorkers = 50
```

Comment:
I think they are used by `genconfig.go` but not `genfallbacks.go`? Nothing important anyway, it's just a command line tool.